src/wolfio.c

```diff
@@ -1409,6 +1409,9 @@ int wolfIO_DecodeUrl(const char* url, int urlSz, char* outName, char* outPath,
             word32 bigPort = 0;
             i = 0;
             cur++;
+
+            XMEMSET(port, 0, sizeof(port));
```

Comment:
This addresses a false positive, but I can live with it, given it's only 6 bytes.  Btw I probably would've resolved it by refactoring that neighborhood to convert the ASCII port to a `word32` without the unnecessary copying of the ASCII.